README.md

```diff
@@ -1,42 +1,76 @@
 # eslint-config-eko
 
-Defines set of Eko Javascript eslint rules based on [Airbnb](https://github.com/airbnb/javascript) eslint rules extended by [Prettier plugin](https://github.com/prettier/eslint-plugin-prettier).
+Defines set of Amity ESLint rules based on [Airbnb](https://github.com/airbnb/javascript) eslint rules extended by [Prettier plugin](https://github.com/prettier/eslint-plugin-prettier).
 
 The package provides 2 sets of rules:
 
-- `eko` - base js rules, import rules
-- `eko/react` - `eko`, react rules, jsx-a11y rules
+- `eko` - eslint, import rules
+- `eko/react` - `eko` + react, react-hook, jsx-a11y rules
 
-## Pre-installation:
+## Installation:
 
-Copy `.prettierrc` file into root of your project. Make sure that you do not change configuration or you will run in a bunch of conflicts between prettier and eslint.
+### Step 1. Mandatory. Copy this configuration to your `.prettierrc` file:
```

Comment:
I think we can add a `.prettierrc` file in here and use this to have inheritance: https://github.com/prettier/prettier/pull/5963